### PR TITLE
tags: created schema for new tags table

### PR DESCRIPTION
**Pull request by jmy-kellogg**

### CODE REVIEW

File: lib/v0/tags/base.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { oneOf } = require('../../helpers/payload-or-null')
```
Comment (qtotuan) [inline lib/v0/tags/base.js]: I don't know how to put a comment on the commit message, but it looks different than the semantic-commit format. Did you use the commit script?

Comment (Endebert) [reply lib/v0/tags/base.js]: also no imperative tense ;)

Comment (jmy-kellogg) [reply lib/v0/tags/base.js]: One person writes a full sentence, a comment, a commit revision to remove one letter. Haha, this feel so much more efficient.

**Pull request merged**